bigframes/_config/bigquery_options.py

```diff
@@ -26,6 +27,12 @@
 import bigframes.constants
 import bigframes.exceptions
 
+
+class OrderingMode(Enum):
```

Comment:
Could we move this to `bigframes.enums`? https://github.com/googleapis/python-bigquery-dataframes/blob/main/bigframes/enums.py

Reply:
Mailed https://github.com/googleapis/python-bigquery-dataframes/pull/870